scripts/paas_app_wrapper.sh

```diff
@@ -51,7 +51,7 @@ case $NOTIFY_APP_NAME in
     ;;
   delivery-worker-save-api-notifications)
     exec scripts/run_app_paas.sh celery -A run_celery.notify_celery worker --loglevel=INFO --concurrency=11 \
-    -Q save-api-email-tasks 2> /dev/null
+    -Q save-api-email-tasks, save-api-sms-tasks 2> /dev/null
```

Comment:
I think I like the fact they go on the same queue. These services aren't ones that are allowed to be worried about slight delays to their messages going out, so it doesn't matter if one service sending out lots of SMS caused some delays to GOV.UK email.

Also it means we only have to scale one worker to control how much DB resource we use by high volume services